doc/rtd/reference/datasources/fallback.rst

```diff
@@ -0,0 +1,19 @@
+.. _datasource_fallback:
+
+Fallback/no datasource
+**********************
+
+This is the fallback datasource when no other datasource can be selected. It
+is the equivalent of an empty datasource, in that it provides an empty string
+as user data, and an empty dictionary as metadata.
+
+It is useful for testing, as well as for occasions when you do not need an
+actual datasource to meet your instance requirements (i.e. you just want to
```

Comment:
It's not so much an issue of styling imo - since "i.e." (id est) stands in for "that is", my usual approach is to replace it with "that is" and read the sentence aloud. If I find myself naturally pausing after "that is", I'll put a comma after i.e. but if I don't then I usually leave it out :)